sdk/src/signer/signers.rs

```diff
@@ -186,4 +186,28 @@ mod tests {
             vec![Signature::default(), Signature::default()],
         );
     }
+
+    #[test]
+    fn test_unsized_signers() {
```

Comment:
nit: since this test is working on the `Transaction` level, can you move it to `transaction/mod.rs`?